client/src/App.vue

```diff
@@ -48,6 +48,16 @@ export default defineComponent({
   },
   watch: {
     $route(to: RouteLocation) {
```

Comment:
Can you link to how this gets wired up like how route get here? Good chance I'm missing something too

Reply:
Done!